src/client/runner/automation/playback/press/utils.js

```diff
@@ -69,3 +70,21 @@ export function getChar (key, shiftModified) {
     return key;
 }
 
+export function getDeepActiveElement (currentDocument) {
+    var doc                   = currentDocument || document;
```

Comment:
I suppose it's obsolete. Because we should have identical approach for same-domain and cross-domain iframes. We can't get activeElement from a cross-domain iframe, so we should do this for a same-domain one. In any case if it'll break something we see it on our functional tests